test/__mocks__/text.mock.ts

```diff
@@ -167,4 +167,5 @@ export const UPDATE_CONFIRMATION_PAGE_TITLE = "Application submitted";
 export const UPDATE_CONFIRMATION_PAGE_REFERENCE_NUMBER = "Your application reference number:";
 export const UPDATE_DUE_DILIGENCE_PAGE_TITLE = "Complete this statement to confirm that verification checks have been completed";
 export const WHO_IS_MAKING_UPDATE_PAGE_TITLE = "Who is completing this update?";
+export const UPDATE_MANAGING_OFFICER_PAGE_TITLE = "Tell us about the individual person";
```

Comment:
Should that not be "Tell us about the managing officer" - possibly with individual. Please ask Matthew Poland.

Reply:
Checked with Matthew, expected tile.

Reply:
Should this not be 'Tell us about the individual person (NOT LIVE)'? (Although i know this is only a temporary title)

Reply:
@bbell-companieshouse There's barely any NOT LIVE suffixes in this file. I think there are 2? So the majority of time we've not added it and I tend to go with the most consistent approach. 

@AndreKramer7654 Barry and I chatted to Matthew Poland during development about the NOT LIVE suffix and his thoughts are that it might not be needed anymore (he got us to add it here for consistency with BO's from K2) and that we can stop adding it. Probably a discussion for refinement.